spec/jmap/push.mdown

```diff
@@ -1,11 +1,16 @@
 # Push
 
-Any modern email client should be able to update instantly whenever the data on the server is changed by another client or message delivery. Push notifications in JMAP occur out-of-band (i.e. not over the same connection as API exchanges) so that they can make use of efficient native push mechanisms on different platforms.
+Push notifications allow clients to efficiently update (almost) instantly to stay in sync with data changes on the server. In JMAP, push notifications occur out-of-band (i.e. not over the same connection as API exchanges), so that they can make use of efficient native push mechanisms on different platforms.
 
 The general model for push is simple and does not send any sensitive data over the push channel, making it suitable for use with less trusted 3rd party intermediaries. The format allows multiple changes to be coalesced into a single push update, and the frequency of pushes to be rate limited by the server. It doesn't matter if some push events are dropped before they reach the client; it will still get all changes next time it syncs.
 
-When something changes on the server, the server pushes a small JSON object to the client with the following property:
+When something changes on the server, the server pushes a JSON object to the client with the following properties:
 
+- **type**: `String`
+  This MUST be the string `"state"`. This is to allow future extension of the
+  push protocol with other event types.
+- **username**: `String`
+  The username of the authenticated user.
```

Comment:
This isn't necessary: a subscription can be created that is bound to a specific context.  (Subscriptions are cheap.)